Formula/btop.rb

```diff
@@ -0,0 +1,45 @@
+class Btop < Formula
+  desc "Resource monitor. C++ version and continuation of bashtop and bpytop"
+  homepage "https://github.com/aristocratos/btop"
+  url "https://github.com/aristocratos/btop/archive/refs/tags/v1.1.2.tar.gz"
+  sha256 "997376596cc6ca808b5a26f9fe25da274c19c2970aaf757943a009f4c28b758c"
+  license "Apache-2.0"
+
+  depends_on "coreutils" => :build
+  depends_on "gcc" => :build
```

Comment:
```suggestion
  fails_with: :clang

  on_macos do
    depends_on "gcc"
  end
```
Why doesn't Clang work? If you need to use GCC, you'll need to keep it around at runtime.

Reply:
This is what upstream suggests. https://github.com/aristocratos/btop#compilation-osx
I have tried it with clang initially, but `-ftree-loop-vectorize -flto=12 -s` options are not supported.

I should add `STATIC=true` to `make` to "make it work" :)

Thanks for `fails_with :clang` magic!

Reply:
Note about `STATIC=true` - upstream suggest to static link to libstdc++/libc++, this is what this flag is for